src/sagemaker/image_uris.py

```diff
@@ -47,6 +46,9 @@ def retrieve(
     model_version=None,
     tolerate_vulnerable_model=False,
     tolerate_deprecated_model=False,
+    sdk_version=None,
+    image_version=None,
```

Comment:
This can then be removed

Reply:
I think we should keep the sdk version, to facilitate the use-case, where someone want to use a prior sdk version, instead of the latest one.

Reply:
image_version is removed